src/source/Main.scala

```diff
@@ -346,6 +346,7 @@ object Main {
       objcppNamespace,
       objcBaseLibIncludePrefix,
       objcSwiftBridgingHeaderWriter,
+      objcSwiftBridgingHeader,
```

Comment:
The variable is declared as `objcSwiftBridgingHeader` inside `main.scala`.
Do you want me to change the name of the property in `generator.scala`  from `objcSwiftBridgingHeaderName` to `objcSwiftBridgingHeader` instead?

Reply:
Oh, I see, it's just the variable name here while `objcSwiftBridgingHeaderName` is the parameter, sorry I got confused.
Yes, I'd prefer to have the same name since this is the case for all the other parameters.